src/transaction.ts

```diff
@@ -93,6 +96,30 @@ export interface RunUpdateCallback {
   (err: null|ServiceError, rowCount: number): void;
 }
 
+export interface CommitRequest {
+  session: string;
+  transactionId?: Uint8Array|string|null;
+  singleUseTransaction?: ProtoITransactionOptions;
+  mutations: ProtoIMutation[];
+}
+
+export interface TransactionSelector {
```

Comment:
Can't we just use `ITransactionSelector`?

Reply:
Same as `ICommitRequest` ⬆️